config/routes.rb

```diff
@@ -131,6 +131,8 @@ def self.matches?(request)
       get ":code", to: "posts#show"
       get ":code/:tab", to: "posts#show", as: "post_tab"
       post "immortalise", to: "posts#immortalise", as: "immortalise_post"
+      patch "archive-transfer", to: "posts#archive_transfer"
```

Comment:
Would it make sense to move this to a `resource` and move `archive_transfer` and `archive_destroy` to a separate controller?

Reply:
Conceptually I always thought of these actions as being on posts with the authorization being a supporting resource, but come to think of it doing it the other way around may make sense. I'll try it out later.